runtime/src/vm/opcode.rs

```diff
@@ -2036,28 +2073,28 @@ pub static OPCODES: Lazy<HashMap<i64, ByteOpcode>> = Lazy::new(|| {
     });
     None
   });
-  unpred_cpu!("eachl", |args, hand_mem, frag, ins_sched| {
-    let arr = hand_mem.get_fractal(args[0]);
+  unpred_cpu!("eachl", |args, mut hand_mem, frag, ins_sched| {
```

Comment:
Don't ask me why Rust can infer that `hand_mem` has to be mutable in all the other opcodes but not here after my refactor. It doesn't make sense to me.

Reply:
oh rust